src/main/groovy/com/github/spotbugs/snom/SpotBugsTask.groovy

```diff
@@ -235,7 +236,7 @@ class SpotBugsTask extends DefaultTask implements VerificationTask {
      * Default value is the compile-scope dependencies of the target sourceSet.
      */
     @InputFiles
```

Comment:
nit: we can remove `@InputFiles` as it's replaced by `@Classpath`

Reply:
That's correct, [unless there is a goal to remain compatible with Gradle < 3.2](https://github.com/gradle/gradle/blob/ee65597ecffcfde800aedda9f4c956b688b956a8/subprojects/core-api/src/main/java/org/gradle/api/tasks/Classpath.java#L38-L39).